pkg/data/components/FieldExistsCondition.yaml

```diff
@@ -2,7 +2,7 @@ kind: FieldExistsCondition
 name: Check if field exists
 style: condition
 type: base
-status: alpha
+status: deprecated
```

Comment:
Since the new one is only in development, if we switch this to deprecated in this PR there wont be a visible component to match this functionality. We should leave this alpha. In a future PR, we'll move all the necessary components to deprecated and all their replacements to alpha at the same time.